### DISCUSSION TITLE
learning codemagic with bitbucket connection

### DISCUSSION THREAD

**Discussion by mehikmat** [codemagic-ci-cd/codemagic-docs#1872] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Error while starting build:

Cannot access the repository https://api.bitbucket.org/2.0/repositories/xxxxxxx
Something went wrong

### Expected results

build success

### Actual results

Cannot access the repository https://api.bitbucket.org/2.0/repositories/xxxxxxxx
Something went wrong

### Build id (optional)

no build id

**Comment by kulakarla** [comment, marked as answer]
Hey @mehikmat 
Have you tried disconnecting the repo and connecting it again? It refreshes the token which should be enough in this case.
